types/flow-example/package.json

```diff
@@ -9,7 +9,7 @@
     "glamor": "^3.0.0-3",
     "glamor-reset": "^3.0.0-2",
     "hoist-non-react-statics": "^2.0.0",
-    "react": "^15.6.1",
+    "react": "^17.0.1",
```

Comment:
It's likely not wise to bump this so that anything between `16.3.1` and `17.0.1` isn't allowed.